src/styles/catalog.scss

```diff
@@ -0,0 +1,123 @@
+.catalog {
+  display: flex;
+  margin: 0 auto;
+  flex-wrap: wrap;
+  width: $catalog-width;
+  padding: 51px 0 0;
+}
+
+.card {
```

Comment:
Card is a separate block according to BEM. It should be in its own file (src/styles/blocks/card.scss)